Mathlib/Logic/Basic.lean

```diff
@@ -697,7 +697,6 @@ protected theorem Decidable.not_forall {p : α → Prop} [Decidable (∃ x, ¬p
    not_forall_of_exists_not⟩
 #align decidable.not_forall Decidable.not_forall
 
-@[simp]
```

Comment:
I don't think this change belongs in this PR; I'm going to revert it and see what breaks.

Reply:
I don't remember changing this file, it might be part of a merge gone wrong?

Reply:
Indeed. That was a mistake on my side.